src/shipping/shipping-in-store-delivery.md

```diff
@@ -0,0 +1,44 @@
+---
+title: In-store Delivery
+---
+
+The In-store delivery method allows to select a source to be used as a pickup location during the checkout of your order.
+
+![]({% link images/images/luma-in-store-example.png %}){: .zoom}
+[_Example of a In-store delivery method during checkout_]({% link sales/checkout-step1-shipping.md %})
+
+## To set up In-store Delivery
+
+The first step is to complete the default settings for In-store Delivery. You can complete this step without changing the scope of the configuration.
```

Comment:
Rewrite this paragraph. 

Enable In-store Delivery and optionally change the default values.

Reply:
done